### PR TITLE
GUAC-1512: Document use of overall concurrency limits within JDBC auth and Docker.

### PR DESCRIPTION
**Pull request by mike-jumper**

### CODE REVIEW

File: src/chapters/docker.xml [hunk 1]
```diff
@@ -209,6 +209,19 @@
                         </thead>
                         <tbody>
                             <row>
+                                <entry><envar>MYSQL_ABSOLUTE_MAX_CONNECTIONS</envar></entry>
+                                <entry>
+                                    <para>The absolute maximum number of concurrent connections to
+                                        allow at any time, regardless of the Guacamole connection or
```
Comment (jmuehlner) [inline src/chapters/docker.xml:215]: Did you mean "allowed"?

File: src/chapters/docker.xml [hunk 2]
```diff
@@ -447,6 +460,19 @@
                         </thead>
                         <tbody>
                             <row>
+                                <entry><envar>POSTGRES_ABSOLUTE_MAX_CONNECTIONS</envar></entry>
+                                <entry>
+                                    <para>The absolute maximum number of concurrent connections to
+                                        allow at any time, regardless of the Guacamole connection or
```
Comment (jmuehlner) [inline src/chapters/docker.xml:466]: Same here

File: src/chapters/docker.xml (hunk 1 above)
Comment (mike-jumper) [inline src/chapters/docker.xml:215]: What's wrong with:

> The absolute maximum number of concurrent connections to allow ...

?

Comment (jmuehlner) [inline src/chapters/docker.xml:215]: Whoops, sorry - misread that.

**Pull request merged**